### DISCUSSION TITLE
Failed to flash arm trusted firmware to Jetson TX2

### DISCUSSION THREAD

**Discussion by Alan-Kuan** [bao-project/bao-hypervisor#118] [General]
Hi,

I followed [the instructions from bao-demos](https://github.com/bao-project/bao-demos/tree/master/platforms/tx2) to install bao on Jetson TX2. However, I stuck at the step to flash arm trusted firmware to Jetson TX2. After the flash command started, it hung for almost an hour. I had nothing to do but interrupt the process.

I am seeking for help to successfully flash the firmware. The followings are some information that might be helpful.

I'm sure the board was in recovery mode.
![image](https://github.com/bao-project/bao-hypervisor/assets/24734750/708f62e4-6dc7-427f-897a-67ed72699cf4)

During the flashing, it showed a message `BootRom is not running`.
![image](https://github.com/bao-project/bao-hypervisor/assets/24734750/54b526c2-9f5e-48b0-98bb-a260ec223af4)

**Comment by danielRep** [comment]
Hi @Alan-Kuan , I'm addressing this issue right now [here](https://github.com/bao-project/bao-demos/issues/51)

**Reply by Alan-Kuan** [reply]
> Hi @Alan-Kuan , I'm addressing this issue right now [here](https://github.com/bao-project/bao-demos/issues/51)

Ok. Thank you!
